packages/next/client/router.ts

```diff
@@ -13,7 +13,7 @@ type SingletonRouterBase = {
   ready(cb: () => any): void
 }
 
-export { Router, RouterContext, NextRouter }
```

Comment:
@timneutkens I don't know why this was reverted. This was how I was able to type my router props before:
```
router: React.ContextType<typeof RouterContext>;
```

Is it available from another location now without having to dig multiple layers deep?